go/run/scripts/test_mixnet.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Testing script for driving the development of the mixnet.
+
+set -o nounset
+set -o errexit
+
+gowhich() {
+	WHICH=$(which which)
+	echo -n "$(PATH="${GOPATH//://bin:}/bin" $WHICH "$1")"
+}
+
+DOMAIN="/tmp/test_domain"
+DOMAIN_PUB="${DOMAIN}.pub"
+TYPE="Soft"
+FAKE_PASS=BogusPass
+
+# Make sure we have sudo privileges before using them to try to start linux_host
+# below.
+sudo test true
+
+
+### Create domain.
+echo "----------------- Creating domain."
```

Comment:
I'd like to keep the style of script output consistent. For the moment that means not having a "----" prefix. I'd be open to a pull request changing that in the future, however.